packages/marshal/docs/smallcaps-cheatsheet.md

```diff
@@ -20,14 +20,20 @@ An example-based summary of the Smallcaps encoding on the OCapN [Abstract Syntax
 | error            | Error         | `TypeError(msg)`      | `{"#error":<msg>,"name":"TypeError"}` |
 
 * The `-0` encoding is defined as above, but not yet implemented in JS.
-* In JS, selectors in transition from symbols to their own representation
+* In JS, passable symbols are in transition from JavaScript symbols to their own representation
 * The number after `"$"` or `"&"` is an index into a separate slots array.
-* Special strings begin with any of the `!"#$%&'()*+,-` characters.
+* ***Special strings*** begin with any of the `!"#$%&'()*+,-` characters.
 * `<expr>` is nested encoding of `expr`.
 * To be passable, arrays, records, and errors must also be hardened.
 * Structs [can only have string-named properties](https://github.com/endojs/endo/blob/master/packages/pass-style/doc/copyRecord-guarantees.md).
 * Errors can also carry an optional `errorId` string property.
 * We expect to expand the optional error properties over time.
 * The ByteArray encoding is not yet designed or implemented.
 
-Every JSON encoding with no special strings anywhere decodes to itself.
+## Readability Invariants
+
+For every JSON encoding with no special strings, the JSON and smallcaps decodings are the same.
+
+For every value with no special strings that round trips through JSON, the JSON and smallcaps encodings are the same.
+
+In other words, for these simple values, ***you can ignore the differences between smallcaps and JSON***.
```

Comment:
I still wish we defined smallcaps as a transformation of passable data into JS objects such that a plain `JSON.stringify` produces the string encoding described above, and that parsing that string with `JSON.parse` is guaranteed to produce a value that is "deep-equal" to the output of the smallcaps transformation.